controllers/common.go

```diff
@@ -550,3 +551,72 @@ func GetCommonRbacRules(privileged bool) []rbacv1.PolicyRule {
 
 	return []rbacv1.PolicyRule{rbacPolicyRule}
 }
+
+// Some frameworks like (e.g., Tobiko and Horizon) require password value to be
+// present in clouds.yaml. This code ensures that we set a default value of
+// 12345678 when password value is missing in the clouds.yaml
+func EnsureCloudsConfigMapExists(
+	ctx context.Context,
+	instance client.Object,
+	helper *helper.Helper,
+	labels map[string]string,
+) (ctrl.Result, error) {
+	const openstackConfigMapName = "openstack-config"
```

Comment:
(blocking) question: I guess this one is not required right? Related as well with @kstrenkova 
comment.
Found TestOperatorCloudsConfigMapName in the volumes.go files.